### PR TITLE
feat(build-env): added builder for python written components

### PR DESCRIPTION
**Pull request by SteBaum**
This PR contains the image of the tdp builder for python written components such as Hue and JupyterLab.

### CODE REVIEW

**Comment by rpignolet** [comment]
Create a folder `build-env-python` outside of `build-env` to avoid sending useless files to docker daemon when building the current image inside `build-env`.

**Comment by SteBaum** [comment]
> Create a folder `build-env-python` outside of `build-env` to avoid sending useless files to docker daemon when building the current image inside `build-env`.

Has been done with last force push.

**Review by rpignolet — APPROVED** [review]

File: build-env-python/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eo pipefail
+
+if [[ -n "$USER_UID" ]] && [[ -n "$USER_GID" ]]; then
+    # Create group and user only if they don't exist
+    [[ ! $(getent group builder) ]] && groupadd -r --gid "$USER_GID" builder
+    if [[ ! $(getent passwd builder) ]]; then
+        useradd --create-home --home-dir /home/builder --uid "$USER_UID" --gid "$USER_GID" --system --shell /bin/bash builder
+    usermod -aG wheel builder
```
Comment (gmndr) [inline build-env-python/docker-entrypoint.sh:9]: It's a detail, but the indentation needs to be corrected here

Comment (SteBaum) [reply build-env-python/docker-entrypoint.sh:9]: Done !

**Review by gmndr — APPROVED** [review]

**Pull request merged**